1648971087497	Sync.LogManager	DEBUG	Flushing file log
1648971087532	Sync.LogManager	DEBUG	Log cleanup threshold time: 1648107087532
1648971087554	Sync.LogManager	DEBUG	Done deleting files.
1648971088214	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: true
1648972025185	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: true
1648972025185	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1648972025189	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1648972025191	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1648972875555	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1648972875556	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1648972875560	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1648972875562	Sync.Service	DEBUG	User-Agent: Firefox/98.0.2 (Linux x86_64) FxSync/1.100.0.20220323001514.desktop
1648972875562	Sync.Service	INFO	Starting sync at 2022-04-03 13:31:15 in browser session AFfLaoP16OVU
1648972875562	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1648972875566	Sync.Status	INFO	Resetting Status.
1648972875567	Sync.Status	DEBUG	Status.service: error.login.failed => success.status_ok
1648972875567	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1648972875567	FirefoxAccounts	DEBUG	getOAuthToken enter
1648972875568	FirefoxAccounts	TRACE	getCachedToken returning cached token
1648972875568	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1648972875568	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1648972875569	Sync.SyncAuthManager	INFO	Getting sync key
1648972875569	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1648972875570	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1648972875570	FirefoxAccounts	DEBUG	getOAuthToken enter
1648972875570	FirefoxAccounts	TRACE	getCachedToken returning cached token
1648972875570	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1648972875570	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1648972875570	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1648972875630	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2153390067 - 
1648972875630	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"<no message>\"  nsresult: \"0x805a1ff3 (<unknown>)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]"}
1648972875757	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2153390067 - 
1648972875757	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:39:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:62:16
_tokenServerExchangeRequest@tokenserverclient.js:241:13
1648972875757	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1648972875757	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1648972875758	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:39:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:62:16
_tokenServerExchangeRequest@tokenserverclient.js:241:13
1648972875759	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1648972875817	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/194602547/info/collections failed: [Exception... "<no message>"  nsresult: "0x805a1ff3 (<unknown>)"  location: "<unknown>"  data: no] No traceback available
1648972875817	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "<no message>"  nsresult: "0x805a1ff3 (<unknown>)"  location: "<unknown>"  data: no] No traceback available
1648972875818	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1648972875818	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 5008916 ms.
1648972875818	Sync.SyncScheduler	DEBUG	Next sync in 5008916 ms. (why=client-backoff-schedule)
1648972875818	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1648972875824	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "<no message>"  nsresult: "0x805a1ff3 (<unknown>)"  location: "<unknown>"  data: no] No traceback available
